DataFormats/EgammaCandidates/src/classes_def.xml

```diff
@@ -321,9 +321,18 @@
   </class>
   <class name="edm::helpers::KeyVal<edm::RefProd<std::vector<reco::Photon> >,edm::RefProd<edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> > > >" />
   <class name="edm::Wrapper<edm::AssociationMap<edm::OneToOne<std::vector<reco::Photon>,edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> >,unsigned int> > >" />
- 
+
   <class name="edm::Ptr<reco::Photon>" />
   <class name="edm::PtrVector<reco::Photon>" />
-  
-</lcgdict>
 
+  <class name="reco::HIPhotonIsolation" ClassVersion="10">
+    <version ClassVersion="10" checksum="1971638020"/>
```

Comment:
I started at 10 because this Twiki page recommended that value:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning